src/V3Options.cpp

```diff
@@ -921,10 +926,25 @@ void V3Options::parseOptsList(FileLine* fl, const string& optdir, int argc, char
                 shift;
                 m_inlineMult = atoi(argv[i]);
             }
+            else if (!strcmp(sw, "-j")) {
+                if ((i+1) >= argc || !isdigit(argv[i+1][0])) {  // No value is given
+                    m_buildJobs = 0;  // Unlimited parallelism
+                }
+                else {
+                    shift;
+                    m_buildJobs = atoi(argv[i]);
+                    if (m_buildJobs <= 0)
+                        fl->v3error("-j accepts positive integer, but "<<argv[i]<<" is passed");
```

Comment:
Nit, I always use {} for multiline statements.

                 if (m_buildJobs <= 0) {
                        fl->v3error("-j accepts positive integer, but " << argv[i] << " is passed");
                 }

Reply:
https://github.com/verilator/verilator/blob/0f96a4fa7dc72668d641ef18d31665edb9c896f7/src/V3Options.cpp#L941-L943